app/src/test/java/com/invisibleteam/goinvisible/mvvm/images/adapter/ImagesItemAdapterTest.java

```diff
@@ -0,0 +1,78 @@
+package com.invisibleteam.goinvisible.mvvm.images.adapter;
+
+import android.view.ViewGroup;
+
+import com.invisibleteam.goinvisible.BuildConfig;
+import com.invisibleteam.goinvisible.model.ImageDetails;
+import com.invisibleteam.goinvisible.mvvm.images.ImagesActivity;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(RobolectricTestRunner.class)
+@Config(constants = BuildConfig.class, sdk = 21)
+public class ImagesItemAdapterTest {
+
+    private ImagesActivity activity;
+    private ImagesItemAdapter imagesItemAdapter;
+
+    @Before
+    public void setUp() {
+        activity = Robolectric.buildActivity(ImagesActivity.class).create().get();
+        imagesItemAdapter = new ImagesItemAdapter();
+    }
+
+    @Test
+    public void whenViewHolderIsCreatedWithNotEmptyList_thenAdapterWillHaveResults() {
+        //given
+        ViewGroup parent = mock(ViewGroup.class);
+        when(parent.getContext()).thenReturn(activity);
+        ImagesItemAdapter.ViewHolder viewHolder = imagesItemAdapter.onCreateViewHolder(parent, 0);
+
+        ImageDetails imageDetails = new ImageDetails("Path", "Name");
+        List<ImageDetails> imagesList = new ArrayList<>();
+        imagesList.add(imageDetails);
```

Comment:
That list could be exposed as member of this class like:
private static final List< ImageDetails > IMAGES_DETAILS_LIST = ImmutableList.of(new ImageDetails("path", "name"));